src/components/AvatarWithIndicator.js

```diff
@@ -10,24 +10,32 @@ const propTypes = {
 
     // url for the avatar
     source: PropTypes.string.isRequired,
+
+    inSettings: PropTypes.bool,
```

Comment:
Rather than tell this component where it is it would make more sense to just tell it how large to be.